packages/analytics-js-common/src/types/IRudderAnalytics.ts

```diff
@@ -76,6 +76,8 @@ export interface IRudderAnalytics<T = any> {
   analyticsInstances: Record<string, T>;
   defaultAnalyticsKey: string;
   logger: ILogger;
+  visitId: string;
+  pageLoadedTimestamp: number;
```

Comment:
We can store it in the state, right?

Reply:
Yes we can, but right now it is not needed. If requires I will add it.

Reply:
If we add it to the state, it'll help us with debugging as well. Please think about it.

Reply:
Addressed